demo/SnowplowDemo/Analytics/Analytics.h

```diff
@@ -0,0 +1,19 @@
+//
```

Comment:
As far as I can tell, this file is not necessary for the demo. Also, there is not much need to separate the tracker code into the `Analytics` folder. Could you please remove this `Analytics.h` file and move the `Tracker.swift` one to the `SnowPlowDemoShareExtension` folder with the rest of the extension code? I think that would make the demo slightly easier to read which could make it more useful for other users.